src/components/webid-form/WebIDForm.jsx

```diff
@@ -0,0 +1,119 @@
+import { QueryEngine } from "@comunica/query-sparql";
+import { getDefaultSession } from "@inrupt/solid-client-authn-browser";
+import { useState, useEffect } from "react";
+import "./WebIDForm.css";
+
+function WebIDForm() {
+  const [errorMessage, setErrorMessage] = useState(undefined);
+  const session = getDefaultSession();
+  const webId = session.info.webId;
+  const isLoggedIn = session.info.isLoggedIn;
+
+  useEffect(() => {}, [webId]);
```

Comment:
What does this do, considering that the function is empty?

Reply:
I'm sorry, it was used before to force render some things but I forgot it there, I have deleted it and other related things in [commit 8f6fef0](https://github.com/SolidLabResearch/solid-web-app-react-template/pull/1/commits/8f6fef091524065fb24cad5693dc707cc43f0e19)